helm/docker-compose.yml

```
@@ -37,35 +34,35 @@ services:
       replicas: 1
     entrypoint: /script/entrypoint.sh --pipeline_script_choice yolov5s.sh
     environment:
-      INPUTSRC: ${RTSP_INPUTSRC}
-      BATCH_SIZE: ${BATCH_SIZE}
-      GST_DEBUG: ${GST_DEBUG}
-      LOG_LEVEL: ${LOG_LEVEL}
-      CLASSIFICATION_OPTIONS: ${CLASSIFICATION_OPTIONS}
-      DETECTION_OPTIONS: ${DETECTION_OPTIONS}
-      OCR_RECLASSIFY_INTERVAL: ${OCR_RECLASSIFY_INTERVAL}
-      BARCODE_RECLASSIFY_INTERVAL: ${BARCODE_RECLASSIFY_INTERVAL}
-      RENDER_MODE: ${RENDER_MODE}
-      PUBLISH: ${PUBLISH}
+      - INPUTSRC=rtsp://camera-simulator:8554/camera_0
+      - BATCH_SIZE=0
+      - GST_DEBUG=0
+      - LOG_LEVEL=ERROR
+      - CLASSIFICATION_OPTIONS=reclassify-interval=1 $DETECTION_OPTIONS
+      - DETECTION_OPTIONS=gpu-throughput-streams=4 nireq=4 batch-size=1
```

I believe the pipeline is running con CPU, thus, this should be like this.
```suggestion
      - DETECTION_OPTIONS=ie-config=CPU_THROUGHPUT_STREAMS=4 nireq=4 batch-size=1
```

---

this change crashes the container - we can re-evaluate and upgrade the dlstreamer version in a future pr.

---

yes, we need to update to dlstreamer 2025